Formula/bpytop.rb

```diff
@@ -1,10 +1,11 @@
 class Bpytop < Formula
   include Language::Python::Virtualenv
+  include Language::Python::Shebang
 
   desc "Linux/OSX/FreeBSD resource monitor"
   homepage "https://github.com/aristocratos/bpytop"
-  url "https://files.pythonhosted.org/packages/e9/1a/fd710c94c0888b139681d132f60608c657ff265b50b9312fd3cba35e08dc/bpytop-1.0.67.tar.gz"
-  sha256 "8b338e3627fa6e991e836bee61ef38988f6a7a3a37dc05c757a92ba4378f95bb"
+  url "https://github.com/aristocratos/bpytop/archive/refs/tags/v1.0.67.tar.gz"
```

Comment:
Can we keep the PyPI source here?

Reply:
It doesn't contain the Makefile, which would mean we couldn't use `make install` like we are right now.

Reply:
We should probably add something like this for `bpytop` in `pypi_formula_mappings.json` then:

https://github.com/Homebrew/homebrew-core/blob/4e3ff58f979e93e5d03a03852eb7e707809ce8a6/pypi_formula_mappings.json#L292

Reply:
I fixed this.

Reply:
Don't we usually prefer the shorter URL:
```suggestion
  url "https://github.com/aristocratos/bpytop/archive/v1.0.67.tar.gz"
```
Probably doesn't really matter as both redirect, but only ~80 out of 2000+ GitHub Formulae use `/refs/tags/` so I wanted to check if we want to keep a particular style.

Otherwise, probably can merge.